commit 86566325d398409637fe920181306d43f5980541
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 7 19:52:04 2010 -0700

    staging: brcm80211: Remove dead code from aiutils.c
    
    Removed unused function ai_view().
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/siutils.h
+++ b/drivers/staging/brcm80211/include/siutils.h
@@ -442,10 +442,6 @@ extern uint32 ai_addrspace(si_t *sih, ui
 extern uint32 ai_addrspacesize(si_t *sih, uint asidx);
 extern void ai_write_wrap_reg(si_t *sih, uint32 offset, uint32 val);
 
-#ifdef BCMDBG
-extern void ai_view(si_t *sih, bool verbose);
-#endif
-
 #ifdef BCMSDIO
 #define si_setcoreidx(sih, idx) sb_setcoreidx(sih, idx)
 #define si_coreid(sih) sb_coreid(sih)
--- a/drivers/staging/brcm80211/util/aiutils.c
+++ b/drivers/staging/brcm80211/util/aiutils.c
@@ -707,93 +707,3 @@ uint32 ai_core_sflags(si_t *sih, uint32
 	return R_REG(sii->osh, &ai->iostatus);
 }
 
-#ifdef BCMDBG
-void ai_view(si_t *sih, bool verbose)
-{
-	si_info_t *sii;
-	osl_t *osh;
-	aidmp_t *ai;
-	uint32 config;
-
-	sii = SI_INFO(sih);
-	ai = sii->curwrap;
-	osh = sii->osh;
-	if (BCM47162_DMP()) {
-		SI_ERROR(("Cannot access mips74k DMP in 47162a0\n"));
-		return;
-	}
-
-	config = R_REG(osh, &ai->config);
-	SI_ERROR(("\nCore ID: 0x%x, config 0x%x\n", si_coreid(&sii->pub),
-		  config));
-
-	if (config & AICFG_RST)
-		SI_ERROR(("resetctrl 0x%x, resetstatus 0x%x, resetreadid 0x%x, resetwriteid 0x%x\n", R_REG(osh, &ai->resetctrl), R_REG(osh, &ai->resetstatus), R_REG(osh, &ai->resetreadid), R_REG(osh, &ai->resetwriteid)));
-
-	if (config & AICFG_IOC)
-		SI_ERROR(("ioctrl 0x%x, width %d\n", R_REG(osh, &ai->ioctrl),
-			  R_REG(osh, &ai->ioctrlwidth)));
-
-	if (config & AICFG_IOS)
-		SI_ERROR(("iostatus 0x%x, width %d\n",
-			  R_REG(osh, &ai->iostatus), R_REG(osh,
-							   &ai->
-							   iostatuswidth)));
-
-	if (config & AICFG_ERRL) {
-		SI_ERROR(("errlogctrl 0x%x, errlogdone 0x%x, errlogstatus 0x%x, intstatus 0x%x\n", R_REG(osh, &ai->errlogctrl), R_REG(osh, &ai->errlogdone), R_REG(osh, &ai->errlogstatus), R_REG(osh, &ai->intstatus)));
-		SI_ERROR(("errlogid 0x%x, errloguser 0x%x, errlogflags 0x%x, errlogaddr " "0x%x/0x%x\n", R_REG(osh, &ai->errlogid), R_REG(osh, &ai->errloguser), R_REG(osh, &ai->errlogflags), R_REG(osh, &ai->errlogaddrhi), R_REG(osh, &ai->errlogaddrlo)));
-	}
-
-	if (verbose && (config & AICFG_OOB)) {
-		SI_ERROR(("oobselina30 0x%x, oobselina74 0x%x\n",
-			  R_REG(osh, &ai->oobselina30), R_REG(osh,
-							      &ai->
-							      oobselina74)));
-		SI_ERROR(("oobselinb30 0x%x, oobselinb74 0x%x\n",
-			  R_REG(osh, &ai->oobselinb30), R_REG(osh,
-							      &ai->
-							      oobselinb74)));
-		SI_ERROR(("oobselinc30 0x%x, oobselinc74 0x%x\n",
-			  R_REG(osh, &ai->oobselinc30), R_REG(osh,
-							      &ai->
-							      oobselinc74)));
-		SI_ERROR(("oobselind30 0x%x, oobselind74 0x%x\n",
-			  R_REG(osh, &ai->oobselind30), R_REG(osh,
-							      &ai->
-							      oobselind74)));
-		SI_ERROR(("oobselouta30 0x%x, oobselouta74 0x%x\n",
-			  R_REG(osh, &ai->oobselouta30), R_REG(osh,
-							       &ai->
-							       oobselouta74)));
-		SI_ERROR(("oobseloutb30 0x%x, oobseloutb74 0x%x\n",
-			  R_REG(osh, &ai->oobseloutb30), R_REG(osh,
-							       &ai->
-							       oobseloutb74)));
-		SI_ERROR(("oobseloutc30 0x%x, oobseloutc74 0x%x\n",
-			  R_REG(osh, &ai->oobseloutc30), R_REG(osh,
-							       &ai->
-							       oobseloutc74)));
-		SI_ERROR(("oobseloutd30 0x%x, oobseloutd74 0x%x\n",
-			  R_REG(osh, &ai->oobseloutd30), R_REG(osh,
-							       &ai->
-							       oobseloutd74)));
-		SI_ERROR(("oobsynca 0x%x, oobseloutaen 0x%x\n",
-			  R_REG(osh, &ai->oobsynca), R_REG(osh,
-							   &ai->oobseloutaen)));
-		SI_ERROR(("oobsyncb 0x%x, oobseloutben 0x%x\n",
-			  R_REG(osh, &ai->oobsyncb), R_REG(osh,
-							   &ai->oobseloutben)));
-		SI_ERROR(("oobsyncc 0x%x, oobseloutcen 0x%x\n",
-			  R_REG(osh, &ai->oobsyncc), R_REG(osh,
-							   &ai->oobseloutcen)));
-		SI_ERROR(("oobsyncd 0x%x, oobseloutden 0x%x\n",
-			  R_REG(osh, &ai->oobsyncd), R_REG(osh,
-							   &ai->oobseloutden)));
-		SI_ERROR(("oobaextwidth 0x%x, oobainwidth 0x%x, oobaoutwidth 0x%x\n", R_REG(osh, &ai->oobaextwidth), R_REG(osh, &ai->oobainwidth), R_REG(osh, &ai->oobaoutwidth)));
-		SI_ERROR(("oobbextwidth 0x%x, oobbinwidth 0x%x, oobboutwidth 0x%x\n", R_REG(osh, &ai->oobbextwidth), R_REG(osh, &ai->oobbinwidth), R_REG(osh, &ai->oobboutwidth)));
-		SI_ERROR(("oobcextwidth 0x%x, oobcinwidth 0x%x, oobcoutwidth 0x%x\n", R_REG(osh, &ai->oobcextwidth), R_REG(osh, &ai->oobcinwidth), R_REG(osh, &ai->oobcoutwidth)));
-		SI_ERROR(("oobdextwidth 0x%x, oobdinwidth 0x%x, oobdoutwidth 0x%x\n", R_REG(osh, &ai->oobdextwidth), R_REG(osh, &ai->oobdinwidth), R_REG(osh, &ai->oobdoutwidth)));
-	}
-}
-#endif				/* BCMDBG */
